podman/domain/config.py

```diff
@@ -1,17 +1,20 @@
 """Read containers.conf file."""
+import sys
 import urllib
 from pathlib import Path
 from typing import Dict, Optional
 
 import xdg.BaseDirectory
 
-try:
-    import toml
-except ImportError:
-    import pytoml as toml
-
 from podman.api import cached_property
 
+if sys.version_info >= (3, 11):
+    from tomllib import loads as toml_loads
+else:
+    try:
+        from tomli import loads as toml_loads
+    except ImportError:
+        from toml import loads as toml_loads
```

Comment:
You may want to remove the sentence about "backwards compatibility" since it's preserved in the end.

Reply:
I could just squash the first two commits, but I'd prefer to keep it like. I want to keep the metadata from the original commit and properly attribute you.